.github/workflows/nglint.yml

```diff
@@ -0,0 +1,45 @@
+name: NgLint
+
+on: 
+  pull_request:
+    branches:
+      - master
+
+
+jobs:
+  nglint:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x]
+
+    steps:
+    - name: Git checkout
+      uses: actions/checkout@v2
+          
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: Npm install and test
+      working-directory: client
+      run: |         
+        npm i
+        npm i -g @angular/cli
+        REPORT=$(ng lint --force=true)
+        echo "$REPORT"
+        echo ::set-env name=EXPOSED_REPORT::"${REPORT//$'\n'/'%0A%0A'}"
+    - name: Comment
```

Comment:
What's that part doing, is it adding the report to Github? and if so is it under the pull request?

Reply:
Line 30 saves the output of the lint to the REPORT variable.
line 31 prints it out so that we can view the output in the CI if we want.
line 32 sets the environment variable EXPOSED_REPORT to the value of REPORT, but with the newline character replaced with %0A%0A. The reason for the switch is that when setting an environment variables (or more broadly when using the github actions toolkit) it will escape the newline character out, effectively deleting them when commenting on the pull request.